.env.example

```
@@ -0,0 +1,17 @@
+# Smartschool credentials
+# Copy this file to .env and fill in your values.
+# .env is gitignored — never commit real credentials.
+
+SMARTSCHOOL_USERNAME=your_username
+SMARTSCHOOL_PASSWORD=your_password
+SMARTSCHOOL_MAIN_URL=your-school.smartschool.be
+SMARTSCHOOL_MFA=YYYY-MM-DD # date of birth (e.g. 2008-07-15) - required if your account uses account verification, leave blank otherwise
+
+# Remote HTTP transport
+MCP_TRANSPORT=streamable-http
+MCP_HOST=0.0.0.0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MCP_HOST key should go before the MCP_TRANSPORT key

(UnorderedKey)